plugwise/smile.py

```diff
@@ -259,7 +259,7 @@ async def request(
         try:
             # Encode to ensure utf8 parsing
             xml = etree.XML(escape_illegal_xml_characters(result).encode())
-        except etree.XMLSyntaxError:
+        except etree.ParseError:
```

Comment:
The only 'real' change between lxml and defusedxml (@CoMPaTech / @bouwew)

Reply:
Good find!